net/net.go

```diff
@@ -622,21 +628,29 @@ func (t *net) Subscribe(ctx context.Context, opts ...core.SubOption) (<-chan cor
 	return channel, nil
 }
 
+func (n *net) ConnectApp(a app.App, threadID thread.ID) (*app.Connector, error) {
```

Comment:
Here's where net implements app.Net. We just need to fetch the thread (can do w/o auth here), and return a connection.

Reply:
Liked my proposed _App_ name, seems to fit quite well for a first idea :P